forward_engineering/services/statements/indexesStatements.js

```diff
@@ -0,0 +1,219 @@
+const _ = require('lodash');
```

Comment:
It's better to import only what's needed from lodash, it will allow tree shaking.